tools/build_updated_packages.rb

```diff
@@ -15,7 +15,9 @@
 
 abort "\nGITLAB_TOKEN environment variable not set.\n".lightred if ENV['GITLAB_TOKEN'].nil?
 abort "\nGITLAB_TOKEN_USERNAME environment variable not set.\n".lightred if ENV['GITLAB_TOKEN_USERNAME'].nil?
+puts "Setting the CREW_AGREE_TIMEOUT_SECONDS environment variable to less than the default of #{CREW_AGREE_TIMEOUT_SECONDS} may speed this up...".orange if ENV['CREW_AGREE_TIMEOUT_SECONDS'].nil?
 
+SKIP_UPDATE_CHECKS = ARGV.include?('--skip')
```

Comment:
Should this be documented somewhere?

Reply:
Done.

Reply:
I mean like in help.   It sorta already is self-documented here.

Reply:
It's only being used in that one script. It's not an addition to `crew` or to `const.rb`.

Reply:
Would it make sense to add to crew eventually?

Reply:
Maybe? If we eventually move the script into crew...

But I don't think it is needed at the moment.